packages/nuxt/src/core/nitro.ts

```diff
@@ -112,10 +112,14 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
       strict: true,
       generateTsConfig: true,
       tsconfigPath: 'tsconfig.server.json',
-      tsConfig: {
-        include: [
-          join(nuxt.options.buildDir, 'types/nitro-nuxt.d.ts')
-        ]
+      typescript: {
+        strict: true,
+        generateTsConfig: true,
+        tsconfigPath: "tsconfig.server.json",
+        tsConfig: {
+          include: [join(nuxt.options.buildDir, "types/nitro-nuxt.d.ts")],
+          exclude: ["../dist", "../.output"]
```

Comment:
We also will need to exclude node_modules if explicitly mentioning exclude paths

Reply:
I'm unfamiliar with the repo. I just saw this bug while developing a nuxt app. You probably can make those exclude paths better :)